modyn/config/schema/downsampling_config.py

```diff
@@ -0,0 +1,145 @@
+from __future__ import annotations
```

Comment:
Can we maybe put that into a `sampling` subdirectory?

Reply:
sure